src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/CodeActions/RazorCodeActionTranslatorManager.ts

```diff
@@ -0,0 +1,38 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import { IRazorCodeActionTranslator } from './IRazorCodeActionTranslator';
+
+export class RazorCodeActionTranslatorManager {
```

Comment:
design nit: This is essentially a composite so I suggest making this a `CompositeCodeActionTranslator` which inherits from `IRazorCodeActionTranslator`. An [example](https://github.com/dotnet/aspnetcore/blob/602cb1dea59751a1297d674cddf7f0729c23e6ca/src/Mvc/Mvc.Core/src/ModelBinding/Validation/CompositeModelValidatorProvider.cs). I feel like the term Manager is too overloaded.